modules/ROOT/pages/salesforce/salesforce-connector-100-studio.adoc

```diff
@@ -221,51 +318,68 @@ keytool -exportcert -alias salesforce-cert -file salesforce-cert.crt -keystore s
 +
 The system exports the public key from the keystore into the workspace. This is the public key that you need to enter in your Salesforce instance.
 +
-. Make sure that you have both the keystore (salesforce-cert.jks) and the public key (salesforce-cert.crt) files in your workspace.
+. Ensure that you have both the keystore (`salesforce-cert.jks`) and the public key (`salesforce-cert.crt`) files in your workspace.
 
 == Use Mutual TLS
 
-In v9.7.0 and later, all authentication types support Mutual TLS. To use this you need a keystore file in the JKS format, and a password for it. See <<Generate a Keystore File>> for more information.
+In v9.7.0 and later, all authentication types support Mutual TLS. To use Mutual TLS, you need a keystore file in the JKS format and a password for the file. See <<Generate a Keystore File>> for more information.
```

Comment:
Should we name the thing that is in version 9.7.0 and later?

Reply:
GLOBALLY change "Mutual TLS" to "mutual TLS."
I looked this up and added it to our Term Presentation List.